lib/Runtime/PlatformAgnostic/Platform/Windows/NumbersUtility.cpp

```diff
@@ -0,0 +1,34 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+#include "RuntimePlatformAgnosticPch.h"
+#include "Common.h"
+#include "ChakraPlatform.h"
+
+namespace PlatformAgnostic
+{
+namespace Numbers
+{
+    size_t Utility::NumberToDefaultLocaleString(const WCHAR *number_string,
+                                                const size_t length,
+                                                WCHAR *buffer,
+                                                const size_t pre_allocated_buffer_size)
+    {
+        size_t count = GetNumberFormatEx(LOCALE_NAME_USER_DEFAULT, 0,
```

Comment:
I don't see a reason to carry an implementation without anything tangible. While deciding for the size_t return type, I have checked the cases and also documentation. On MSDN it clearly says the return value is >= 0.